tpm2/test/tpm2_test.go

```diff
@@ -2080,3 +2080,29 @@ func TestDictionaryAttackLockReset(t *testing.T) {
 		t.Fatalf("got %d, expected 0", caps[0].(TaggedProperty).Value)
 	}
 }
+
+func TestTPMLDigest(t *testing.T) {
```

Comment:
This test case seems like it may belong in encoding_test.go instead. This test doesn't call out to the TPM.

Reply:
agreed